core/types/payload/cancelproducer.go

```diff
@@ -9,63 +9,84 @@ import (
 	"github.com/elastos/Elastos.ELA/crypto"
 )
 
-const CancelProducerVersion byte = 0x00
+type ProducerOperation byte
 
-type CancelProducer struct {
+const (
+	ProducerOperationVersion byte = 0x00
+
+	OperationCancel   ProducerOperation = 0x01
```

Comment:
Flag definition often use short name as prefix. For example ServiceFlag -> SF,  SFNodeNetwork, SFNodeBloom.
So OperationCancel can be POCancel.